camptix.php

```diff
@@ -470,9 +470,18 @@ function enqueue_scripts() {
 			'enterEmail' => __( 'Please enter the e-mail addresses in the forms above.', 'camptix' ),
 			'ajaxURL'    => admin_url( 'admin-ajax.php' ),
 		) );
-
 		// Let's play by the rules and print this in the <head> section.
 		wp_enqueue_style( 'camptix' );
+
+
+		if ( (bool) $this->options['select2_enabled'] ) {
+			wp_register_style( 'select2', plugins_url( '/external/select2/css/select2.min.css', __FILE__ ), array(), $this->version );
```

Comment:
The version numbers used to register the stylesheet and script should be the version of Select2, not CampTix.